### ISSUE TITLE
Update in Fdroid-Market

### ISSUE THREAD

**Issue by jobbb2** [metinkale38/prayer-times-android#178]
Please can you sometimes update your App in Fdroid-Market. In Google Playstore you have an newer version.

**Comment by Farzat07** [comment]
I second that. Can't really trust google on this.

**Comment by light700** [comment]
Yes please, it's important to have f-droid version updated. I thought it's done automatically but for some reason it's not. I'm not getting prayer time anymore from the app. Please brother :)

**Comment by mbnoimi** [comment]
+1
